pyqtgraph/imageview/ImageView.py

```diff
@@ -483,12 +483,12 @@ def evalKeyState(self):
             if key == QtCore.Qt.Key.Key_Right:
                 self.play(20)
                 self.jumpFrames(1)
-                self.lastPlayTime = ptime.time() + 0.2  ## 2ms wait before start
+                self.lastPlayTime = perf_counter() + 0.2  ## 2ms wait before start
```

Comment:
Is this 200 ms?

Reply:
yeah, I'm thinking that the comment here is wrong...I should look at the git history to try and get a little more context.

Reply:
@outofculture you have insight here?  Looks like this manipulates the timing during key-strokes, I'm assuming the comment is wrong, but should the value being added be modified?

Reply:
This very much looks like an outdated comment. If we can't figure it out, maybe just drop the "2ms" value and keep it as  `# wait before start` ?

Reply:
I'll give @outofculture a day or so to chime in but yeah otherwise drop the numerical value in the comment I think is the best way to go forward.

Reply:
This comment is from a Luke-omnibus commit 10 years ago, and it's definitely incorrect compared to the code. `lastPlayTime` is only used in the `ImageView.timeout` method, which controls the auto-advancing play behavior and has a no-op when "`lastPlayTime` is in the future". If I had to give this line a comment, it would be "effectively pauses playback for 0.2s".